redirects.json

```diff
@@ -180,6 +180,22 @@
       "source": "/:locale/docs/guides/timers-in-node/",
       "destination": "/:locale/learn/asynchronous-work/discover-javascript-timers"
     },
+    {
+      "source": "/:locale/docs/guides/debugging-getting-started/",
+      "destination": "/:locale/learn/getting-started/debugging"
+    },
```

Comment:
This is already redirected earlier in the file (via https://github.com/nodejs/nodejs.org/pull/6290).

Reply:
Ok, I'll remove it but i'll wait if there is any other comments

Reply:
You can remove it as Richard mentioned :)